src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -220,8 +221,13 @@ export const TransactionDetailsSummaryItem: FC<TransactionDetailsSummaryItemProp
         data-test="buyerTotalDisplayAmount"
       />
       <Spacer y={2} />
+      {offer?.shippingTotal ? (
+        <Text variant="sm" color="black60">
+          *Estimate Only. Price may vary once offer is finalized.
+        </Text>
+      ) : null}
       <Text variant="sm" color="black60">
-        *Additional duties and taxes{" "}
+        † Additional duties and taxes{" "}
```

Comment:
Can't use `<sup>` here, and from what I've seen I'd have to use `lineHeight` instead but not working...

Reply:
Palette has a `<Sup>` component that should work for us here
https://palette.artsy.net/elements/sup/#gatsby-focus-wrapper

If we do something like:
```
<Sup>†</Sup>Additional duties and taxes{" "}
```
^ it should evaluate as Superscript